packages/tempus/src/Tempus_StepperDIRK_impl.hpp

```diff
@@ -55,7 +55,7 @@ void StepperDIRK<Scalar>::setup(
   this->setUseEmbedded(        useEmbedded);
   this->setZeroInitialGuess(   zeroInitialGuess);
 
-  this->setStageNumber(0);
+  this->setStageNumber(-1);
```

Comment:
This is a good idea. Is there any scheme where -1 would be valid? We could use std::invalid as an alternative here.

Reply:
All RK schemes are multistage ( s >= 1 ) methods. And we can consider BDF schemes a single stage ( s = 1 ), multi-step schemes ( k >= 1 ). So to my knowledge, I don't think any scheme can have a valid s = -1.